### PR TITLE
Editorial: Clarify PlainYearMonth uses integer mathematical values internally

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: spec/plainyearmonth.html [hunk 1]
```diff
@@ -27,9 +27,12 @@ <h1>Temporal.PlainYearMonth ( _isoYear_, _isoMonth_ [ , _calendarLike_ [ , _refe
         1. If _referenceISODay_ is *undefined*, then
           1. Set _referenceISODay_ to *1*<sub>𝔽</sub>.
         1. Let _y_ be ? ToIntegerOrInfinity(_isoYear_).
+        1. If _y_ is +∞ or -∞, throw a *RangeError* exception.
```
Comment (ptomato) [inline spec/plainyearmonth.html:30]: Extreme nitpick: Unicode minus sign
```suggestion
        1. If _y_ is +∞ or −∞, throw a *RangeError* exception.
```
(feel free to disregard, we can always fix these up later)

File: polyfill/test/PlainYearMonth/constructor/constructor/infinity-throws-rangeerror.js [hunk 2]
```diff
@@ -13,17 +14,33 @@ assert.throws(RangeError, () => new Temporal.PlainYearMonth(Infinity, 1));
...
+    "infinite year",
+    [O(Infinity, "year"), O(1, "month"), () => "iso8601", O(1, "day")],
+    ["get year.valueOf", "call year.valueOf"]
+  ],
+  [
+    "infinite month",
+    [O(1970, "year"), O(Infinity, "month"), () => "iso8601", O(1, "day")],
+    ["get year.valueOf", "call year.valueOf", "get month.valueOf", "call month.valueOf"]
+  ],
+  [
```
Comment (ptomato) [inline polyfill/test/PlainYearMonth/constructor/constructor/infinity-throws-rangeerror.js:29]: This invalid-calendar test is a bit off-topic for this test file, though I could also see the reasoning for including it here. It's essentially duplicated in the negative-infinity test though. I don't feel strongly about it.

**Review by ptomato — APPROVED** [review]
:+1:

File: spec/plainyearmonth.html (hunk 1 above)
Comment (ljharb) [reply spec/plainyearmonth.html:30]: i'm pretty sure we use a normal en dash for negation in the spec

**Pull request merged**